diff --git a/node_modules/@firebase/messaging/dist/esm/index.esm.js b/node_modules/@firebase/messaging/dist/esm/index.esm.js
index 25417c4..0216082 100644
--- a/node_modules/@firebase/messaging/dist/esm/index.esm.js
+++ b/node_modules/@firebase/messaging/dist/esm/index.esm.js
@@ -756,7 +756,7 @@ function getPushSubscription(swRegistration, vapidKey) {
                         return [2 /*return*/, subscription];
                     }
                     return [2 /*return*/, swRegistration.pushManager.subscribe({
-                            userVisibleOnly: true,
+                            userVisibleOnly: false,
                             // Chrome <= 75 doesn't support base64-encoded VAPID key. For backward compatibility, VAPID key
                             // submitted to pushManager#subscribe must be of type Uint8Array.
                             applicationServerKey: base64ToArray(vapidKey)
diff --git a/node_modules/@firebase/messaging/dist/esm/index.esm2017.js b/node_modules/@firebase/messaging/dist/esm/index.esm2017.js
index e350f58..873c3db 100644
--- a/node_modules/@firebase/messaging/dist/esm/index.esm2017.js
+++ b/node_modules/@firebase/messaging/dist/esm/index.esm2017.js
@@ -565,7 +565,7 @@ async function getPushSubscription(swRegistration, vapidKey) {
         return subscription;
     }
     return swRegistration.pushManager.subscribe({
-        userVisibleOnly: true,
+        userVisibleOnly: false,
         // Chrome <= 75 doesn't support base64-encoded VAPID key. For backward compatibility, VAPID key
         // submitted to pushManager#subscribe must be of type Uint8Array.
         applicationServerKey: base64ToArray(vapidKey)
diff --git a/node_modules/@firebase/messaging/dist/esm/index.sw.esm2017.js b/node_modules/@firebase/messaging/dist/esm/index.sw.esm2017.js
index 9ef5e5e..a838009 100644
--- a/node_modules/@firebase/messaging/dist/esm/index.sw.esm2017.js
+++ b/node_modules/@firebase/messaging/dist/esm/index.sw.esm2017.js
@@ -564,7 +564,7 @@ async function getPushSubscription(swRegistration, vapidKey) {
         return subscription;
     }
     return swRegistration.pushManager.subscribe({
-        userVisibleOnly: true,
+        userVisibleOnly: false,
         // Chrome <= 75 doesn't support base64-encoded VAPID key. For backward compatibility, VAPID key
         // submitted to pushManager#subscribe must be of type Uint8Array.
         applicationServerKey: base64ToArray(vapidKey)
diff --git a/node_modules/@firebase/messaging/dist/index.cjs.js b/node_modules/@firebase/messaging/dist/index.cjs.js
index c580710..af4fd8c 100644
--- a/node_modules/@firebase/messaging/dist/index.cjs.js
+++ b/node_modules/@firebase/messaging/dist/index.cjs.js
@@ -760,7 +760,7 @@ function getPushSubscription(swRegistration, vapidKey) {
                         return [2 /*return*/, subscription];
                     }
                     return [2 /*return*/, swRegistration.pushManager.subscribe({
-                            userVisibleOnly: true,
+                            userVisibleOnly: false,
                             // Chrome <= 75 doesn't support base64-encoded VAPID key. For backward compatibility, VAPID key
                             // submitted to pushManager#subscribe must be of type Uint8Array.
                             applicationServerKey: base64ToArray(vapidKey)
diff --git a/node_modules/@firebase/messaging/dist/index.sw.cjs b/node_modules/@firebase/messaging/dist/index.sw.cjs
index 905fe73..a041165 100644
--- a/node_modules/@firebase/messaging/dist/index.sw.cjs
+++ b/node_modules/@firebase/messaging/dist/index.sw.cjs
@@ -759,7 +759,7 @@ function getPushSubscription(swRegistration, vapidKey) {
                         return [2 /*return*/, subscription];
                     }
                     return [2 /*return*/, swRegistration.pushManager.subscribe({
-                            userVisibleOnly: true,
+                            userVisibleOnly: false,
                             // Chrome <= 75 doesn't support base64-encoded VAPID key. For backward compatibility, VAPID key
                             // submitted to pushManager#subscribe must be of type Uint8Array.
                             applicationServerKey: base64ToArray(vapidKey)
